extensions/azurePublishNew/src/node/azureResources/servicePlan.ts

```diff
@@ -5,40 +5,31 @@ import { TokenCredentials } from '@azure/ms-rest-js';
 import { WebSiteManagementClient } from '@azure/arm-appservice';
 
 import { parseRuntimeKey } from '../../../../../Composer/packages/lib/shared';
-import {
-  ProvisionConfig,
-  ProvisionWorkingSet,
-  ResourceConfig,
-  ResourceDefinition,
-  ResourceProvisionService,
-} from '../types';
+import { ProvisionConfig, ProvisionWorkingSet, ResourceDefinition, ResourceProvisionService } from '../types';
 import { createCustomizeError, ProvisionErrors } from '../../../../azurePublish/src/node/utils/errorHandler';
+import { AzureResourceTypes } from '../constants';
+import { ServicePlanResourceConfig } from '../availableResources';
```

Comment:
This import will cause a cycle because availableResources depends on this file.

Reply:
fixed